runtime/test/manifest-test.js

```diff
@@ -281,5 +276,43 @@ describe('manifest', function() {
     };
     let manifest = await Manifest.load('somewhere/a', loader, registry);
     assert(registry['somewhere/a path/b']);
-  })
+  });
+  it('can parse a manifest containing a recipe with slots', async () => {
+    let manifest = await Manifest.parse(`
+      recipe SomeRecipe
+        map #someView1 as myView
+        SomeParticle
+          someParam -> myView
+          consume mySlot
+          provide otherSlot (someParam) as slot0
```

Comment:
I think we will still want to iterate on this syntax (later) to make the view constraints a bit more readable.

Perhaps something like:

```
SomeParticle
  provide otherSlot
    display someParam
```